go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/honeycombio/dynsampler-go v0.6.3
 	github.com/honeycombio/hpsf v0.5.1-0.20250605195630-9b072f9dbd61
-	github.com/honeycombio/husky v0.36.0
+	github.com/honeycombio/husky v0.37.1-0.20250730154742-ddf9db5b5fe7
```

Comment:
We can update this to point to the latest commit from `main` now

Reply:
That's what this is.